lib/zendesk_apps_tools/translate.rb

```diff
@@ -95,6 +95,14 @@ def self.source_root
     end
 
     no_commands do
+      def fetch_locale_async(locale, app_package)
+        Thread.new do
+          say("Fetching #{locale['locale']}")
```

Comment:
Ah cool. Thor must sync the output somewhere.
On 20 Apr 2016 6:57 pm, "Olaf Kwant" <notifications@github.com> wrote:

> In lib/zendesk_apps_tools/translate.rb
> <https://github.com/zendesk/zendesk_apps_tools/pull/119#discussion_r60372453>
> :
>
> > @@ -95,6 +95,14 @@ def self.source_root
> >      end
> >
> >      no_commands do
> > +      def fetch_locale_async(locale, app_package)
> > +        Thread.new do
> > +          say("Fetching #{locale['locale']}")
>
> @catkins <https://github.com/catkins> The output looks good btw
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/zendesk_apps_tools/pull/119/files/ba9072876e57675b5196b681ae136b0cf910c4f7#r60372453>
>